File: greedy/change.cpp
#include <iostream>
using namespace std;

int n = 1260;
int count = 0;

int coinTypes[4] = {500, 100, 50, 10};

int main()
{
    for (int i = 0; i < 4; i++)
    {
        int coin = coinTypes[i];
        count = count + (n / coin);
        n = n % coin;
    }
    cout << count << endl;
    return 0;
}

File: greedy/change/change.py
n = 1260  # 잔돈
count = 0  # 동전의 개수

coin_types = [500, 100, 50, 10]  # 동전의 종류

for coin in coin_types:
    count += n // coin  # 동전의 개수 계산
    n %= coin  # 남은 잔돈

print(count)
